doc/site/guides/lua-language-server.markdown

```diff
@@ -0,0 +1,227 @@
+# Lua Language Server
```

Comment:
You need to add a header if you want to make this article available on the site. Add yourself to `doc/site/_authors` and something like this on the top of this file (with the proper adjustments):

```
---
layout: post
title: Choose Recoil
parent: Guides
permalink: guides/choose-recoil
author: sprunk
---
```